### PR TITLE
add chords for two rus songs

### PR DESCRIPTION
**Pull request by jacobvr**
- https://www.gfcsongs.com/song/iisus-spasibo
- https://www.gfcsongs.com/song/silnyi-dobryi-bog

### CODE REVIEW

File: rGFC/text/Иисус, Спасибо.txt [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+Прол[Em]ив на нём свят[C]ую кр[D]овь
+Ты п[Em]оражён Отц[C]ом был, чтоб я ж[G]изнь им[D]ел
+И ж[C]изнь отдал, мой грех взяв на С[D]ебя
+
+Мой гр[G]ех ом[D]ыла Твоя кр[Em]овь
+Иисус, спас[C]ибо
+Крест[G]ом доказан[D]а Отца люб[Em]овь
+Иисус, спас[C]ибо
+И б[Em]ывшего враг[D]а за ст[G]ол Свой посад[C]ил
+Иисус, спас[D]ибо[G]
```
Comment (maxned) [inline rGFC/text/Иисус, Спасибо.txt:13]:
```suggestion
Ты т[Em]айну нам откр[C]ыл через тяж[G/B]ёлый кр[D]ест
Прол[C]ив на нём свят[G/B]ую кр[D]овь
Ты п[Em]оражён Отц[C]ом был, чтоб я ж[G/B]изнь им[D]ел
И ж[Am]изнь отдал, мой гр]G/B]ех взяв на С[D]ебя

Мой гр[G]ех ом[D]ыла Твоя кр[Em]овь
Иисус, спас[C]ибо
Крест[G]ом доказан[D]а Отца люб[Em]овь
Иисус, спас[C]ибо
И б[Em]ывшего враг[D/F#]а за ст[G]ол Свой посад[C]ил
Иисус, спас[D]ибо[G][C][D]
```

File: rGFC/text/Иисус, Спасибо.txt [hunk 2]
```diff
@@ -25,8 +25,8 @@ Title: Иисус, Спасибо
 Иисус, спасибо
 2x
 
-Иисус Христос
-Жить буду для Тебя
+[Em]Иис[D]ус Христ[G]ос
+Жить буду [Em]для Т[D]ебя[C]
```
Comment (maxned) [inline rGFC/text/Иисус, Спасибо.txt:29]:
```suggestion
[Em]Иис[D/F#]ус Христ[G]ос
Ж[C]ить буду [G/B]для Т[D]ебя[C]
```

**Review by maxned — CHANGES_REQUESTED** [review]
I suggest the first one to be like we do it at church. I noticed the chords were quite different.

File: rGFC/text/Иисус, Спасибо.txt (hunk 1 above)
Comment (jacobvr) [reply rGFC/text/Иисус, Спасибо.txt]: Nice! Totally! I will update a few things, though
- `гр]G/B]ех` is broken
- `[G][C][D]` will be smashed together
Let me update that

**Review by maxned — APPROVED** [review]

**Pull request merged**